### PR TITLE
[Feature][Branch] Add AutoComplete Render Options

### PR DESCRIPTION
**Pull request by imjihun**
## Description

- add background highlight when touched
- set value for Render Input when selected

## Related Issues

#50 

## Tests

none yet.

## Checklist

- [x] I read the [Contributor Guide](https://github.com/dooboolab/dooboo-ui-native/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] Run `yarn test` or `yarn test -u` if you need to update snapshot.
- [x] Run `yarn lint`
- [x] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

**Comment by hyochan** [comment]
@marsinearth Please review the beautiful work given.

File: src/components/shared/AutoComplete/index.tsx [hunk 1]
```diff
@@ -51,8 +52,12 @@ export default function AutoComplete({
     toggleOptions((prevStatus) => !prevStatus);
   }, []);
 
-  const onPressOption = useCallback((label: string) => {
-    setInnerValue(label);
+  const onPressOption = useCallback((data: DummyDatum) => {
+    if (data && data.label) {
+      setInnerValue(data.label);
+    }
+    setSelectedData(data);
```
Comment (marsinearth) [inline src/components/shared/AutoComplete/index.tsx:60]: it seems `const [selectedData, setSelectedData] = useState<DummyDatum>();` can substitute `const [innerValue, setInnerValue] = useState<string>(value);` completely. Can you manage the related logic?

**Review by marsinearth — CHANGES_REQUESTED** [review]
please make modifications

**Comment by marsinearth** [comment]
circle-ci-ts-failure
and please solve this Circle CI typescript error, too.

**Review by marsinearth — APPROVED** [review]

**Pull request merged**